js/lib/utils/ItemList.js

```diff
@@ -21,6 +21,21 @@ export default class ItemList {
   }
 
   /**
+   * Check whether the list is empty.
+   *
+   * @returns {boolean}
+   * @public
+   */
+  isEmpty() {
+    for(var prop in this.items) {
```

Comment:
Probaly should use the variable name `i` [for consistency](https://github.com/zinsserzh/core/blob/f6810675437ff4605d991e098d631907b4d2a94d/js/lib/utils/ItemList.js#L109)